fdbserver/QuietDatabase.actor.cpp

```diff
@@ -586,7 +586,7 @@ ACTOR Future<Void> waitForQuietDatabase(Database cx,
                                         int64_t maxPoppedVersionLag = 30e6) {
 	state Future<Void> reconfig =
 	    reconfigureAfter(cx, 100 + (deterministicRandom()->random01() * 100), dbInfo, "QuietDatabase");
-
+	state Future<Void> disableWiggling = setPerpetualStorageWiggle(cx, LiteralStringRef("0"));
```

Comment:
this can fail but you never wait on the future -- therefore in some scenarios quietDatabase might still fail. I think you will need to wait on this future. Otherwise, if this throws you will not see the error but instead observe a quietDatabase failure which is a bad signal to debug